src/gui/PageViewFindObjectHelper.h

```diff
@@ -162,7 +162,11 @@ class PlayObject : public BaseSelectObject
 
 			if (!fn.empty())
 			{
-				view->getXournal()->getControl()->getAudioController()->startPlayback(Path::fromUri(view->settings->getAudioFolder()).str() + "/" + fn, (unsigned int) ts);
+				if (fn.rfind('/', 0) != 0)
```

Comment:
Windows use Backslash, may use the Path class instead.Or is the Path an internal one?